plugins/lsp/language-servers/default.nix

```diff
@@ -410,6 +410,12 @@ let
         '';
       };
     }
+    {
+      name = "openscad-lsp";
+      description = "A Language Server Protocol server for OpenSCAD";
+      package = pkgs.openscad-lsp;
```

Comment:
```suggestion
```
Not needed.

Reply:
(`mkLsp` defaults to using `pkgs.${name}`)